.github/dependabot.yml

```diff
@@ -6,3 +6,7 @@ updates:
       interval: "daily"
     commit-message:
       prefix: chore
+  - package_manager: "docker"
```

Comment:
This should be `package-ecosystem`

https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/configuration-options-for-dependency-updates#about-the-dependabotyml-file

Reply:
my bad.